frontend/packages/console-dynamic-plugin-sdk/src/constants.ts

```diff
@@ -1,3 +1,36 @@
 export const extensionsFile = 'console-extensions.json';
 export const pluginManifestFile = 'plugin-manifest.json';
 export const remoteEntryFile = 'plugin-entry.js';
+export const featureReducerName = 'FLAGS';
+
+export enum FLAGS {
+  AUTH_ENABLED = 'AUTH_ENABLED',
+  PROMETHEUS = 'PROMETHEUS',
+  CHARGEBACK = 'CHARGEBACK',
```

Comment:
@spadgett there are usages of the flags you wish to delete, I think deleting flags should be in seperate PRs with their usages

Reply:
The problem is if we forget to do that before 4.9 ships, the flag is part of the API and we can't remove it without making an API breaking change. It would be better to make it a separate constant somewhere and not part of the public enum.

Chargeback we can't remove from the UI yet since it's only deprecated, not removed.

The service catalog code we should remove from the repo. That can be a separate PR.